hackathon_site/event/jinja2/event/landing.html

```diff
@@ -27,292 +29,296 @@
-                <a href="https://www.google.com/" class="col s2 center" rel="noopener" target="_blank"><img src="{{ static('event/images/sponsors/sponsor.svg') }}" alt="sponsor-name"/></a>
-                <br />
+
+                <!-- Partners -->
+                <a href="https://www.ecobee.com/en-ca/" class="col s2 offset-s3" rel="noopener" target="_blank"><img src="{{ static('event/images/sponsors/ecobee_white.svg') }}" alt="ecobee" /></a>
+                <a href="https://www.amd.com/" class="col s2" rel="noopener" target="_blank"><img src="{{ static('event/images/sponsors/AMD_white.svg') }}" alt="AMD" /></a>
+
+                <!-- TBD -->
+                <a href="https://www.huawei.com/ca/" class="col s2" rel="noopener" target="_blank"><img src="{{ static('event/images/sponsors/Huawei_white.svg') }}" alt="Huawei" /></a>
             </div>
-            <p class="center">
+            <p class="center white-text">
                 If you would like to sponsor us, feel free to reach out:
-                <a href="mailto:hello@email.com" class="hoverLink">hello@email.com</a>
+                <a href="mailto:newhacks@ieee.utoronto.ca" class="hoverLink lightText">newhacks@ieee.utoronto.ca</a>
             </p>
         </div>
     </div>
-</div>
-<div class="container">
-    <div class="section">
+    <div class="container section">
         <div class="s12 m8 flexColCenter">
-            <h3 class="center">Check out last year!</h3>
-
-            <video class="video center" controls>
-                <source src="https://file-examples-com.github.io/uploads/2017/04/file_example_MP4_1280_10MG.mp4" type="video/mp4" />
-                Your browser does not support the video tag.
-            </video>
+            <h3 class="center white-text">Check out last year!</h3>
+            <iframe class="video center" src="https://www.youtube.com/embed/Vfi3pRkoyOI" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
         </div>
     </div>
     <br /><br />
 </div>
-
 <div class="carousel">
-    <!-- IMAGES MUST ALL BE THE SAME SIZE -->
-    <a class="carousel-item" ><img src="https://i.imgur.com/UoVt0SD.jpg" alt="2019-image-1"/></a>
-    <a class="carousel-item" ><img src="https://i.imgur.com/tFJUdpY.jpg" alt="2019-image-2"/></a>
-    <a class="carousel-item" ><img src="https://i.imgur.com/EeFW9yt.jpg" alt="2019-image-3"/></a>
-    <a class="carousel-item" ><img src="https://i.imgur.com/ho1TpX3.jpg" alt="2019-image-4"/></a>
-    <a class="carousel-item" ><img src="https://i.imgur.com/NuVyjXK.jpg" alt="2019-image-5"/></a>
-    <a class="carousel-item" ><img src="https://i.imgur.com/r7ZMttJ.jpg" alt="2019-image-6"/></a>
+    <a class="carousel-item" href="#one!"><img src="https://i.imgur.com/leuHG62.jpg" alt="2019-image-1" /></a>
```

Comment:
Are you sure these hrefs needed to be added back? Seems to work without them, and it's strange to have them in the browser hash if you click an image

Reply:
They dont work for me when I remove the href